Content.Server/Fluids/EntitySystems/SpillableSystem.cs

```diff
@@ -35,6 +37,31 @@ public override void Initialize()
         base.Initialize();
         SubscribeLocalEvent<SpillableComponent, LandEvent>(SpillOnLand);
         SubscribeLocalEvent<SpillableComponent, GetOtherVerbsEvent>(AddSpillVerb);
+        SubscribeLocalEvent<SpillableComponent, GotEquippedEvent>(OnGotEquipped);
+    }
+
+    private void OnGotEquipped(EntityUid uid, SpillableComponent component, GotEquippedEvent args)
+    {
+        if (!component.SpillWorn)
+            return;
+
+        if (!EntityManager.TryGetComponent(uid, out ClothingComponent? clothing))
+            return;
```

Comment:
FIY: have you heard of our lord and saviour: entity system proxy methods? Not really necessary, just makes code a little nicer to read with entity-manager function abbreviations. I guess this also applies to the xenoarchaeology pr if you didn't use them there.
```suggestion
        if (!TryComp(uid, out ClothingComponent? clothing))
            return;
```

Reply:
Nice. Thanks, I will use that in future PRs.